torch/utils/data/datapipes/utils/common.py

```diff
@@ -104,6 +104,23 @@ def validate_pathname_binary_tuple(data: Tuple[str, IOBase]):
         )
 
 
+# Deprecated function names and its corresponding kwargs for the `_deprecation_warning` function
+_deprecated_functional_names: Dict[str, Dict] = {"open_file_by_fsspec":
```

Comment:
Going forward, we might need `_deprecated_functional_names` for both `IterDataPipe` and `MapDataPipe`.

Reply:
I'm going to add a string to indicate whether it is for `IterDataPipe` or `MapDataPipe`.